### DISCUSSION TITLE
Why modal open toggles style attribute instead of using utility classes?

### DISCUSSION THREAD

**Discussion by Jimmys20** [twbs/bootstrap#38036] [Q&A]
Hello, this might be a silly question but why does the Modal component use `style="display: none;"` and `style="display: block;"` when it's closed or open instead of the using bootstrap utility classes `d-none` and `d-block`?

**Comment by ExtravagantBoost** [comment]
because the utility classes are for the user to use, if you want to change this so that it doesnt use the style attribute, edit the JS code controlling this, though `display:none` is equivalent to `d-none` and `d-block` is equivalent to `display:block`, so changing the code wont do anything to impact the function of a modal
